### PR TITLE
Bayck dolphin

### PR DESCRIPTION
**Pull request by BAYCKRC**
Modifying Partial Configuration of 'BAYCK-DOLPHIN'

### CODE REVIEW

**Comment by nerdCopter** [comment]
please resolve conflict.  may have to rebase on master.

File: configs/BAYCK_DOLPHIN/config.h [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_MPU6500
+#define USE_ACC_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_SPI_BMP280
+
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_FLASH_W25N01G
```
Comment (haslinghuis) [inline configs/BAYCK_DOLPHIN/config.h:44]: Please try avoiding this chip due to bad performance.

Comment (BAYCKRC) [reply configs/BAYCK_DOLPHIN/config.h:44]: FLASH_W25N01G，What happened causing poor performance?

**Comment by nerdCopter** [comment]
prefer not repeatedly closing and re-opening new PR's. prefer just fix the PR.

**Pull request closed without merging**